api/libproject/cloudsqlproxycontext.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "../../libproject/utils/common"
+require_relative "environments"
+
+class CloudSqlProxyContext
+  attr_reader :env_file_path
+
+  def initialize(env_key, gcloud_context)
+    raise ArgumentError("Invalid GcloudContext") unless gcloud_context.account
+    @bucket_name = Environments::CREDENTIALS_BUCKET_NAMES.fetch(env_key)
+    @bucket_file_name = Environments::DB_VARS_FILE_NAMES.fetch(env_key)
+    @env_file_path = "db/vars.#{env_key}.env"
+  end
+
+  def run()
```

Comment:
In the case of this class, a run block makes sense since it has to do some cleanup in the end.